hyperspy/axes.py

```diff
@@ -210,6 +210,7 @@ class DataAxis(t.HasTraits, UnitConversion):
     high_index = t.Int()
     slice = t.Instance(slice)
     navigate = t.Bool(t.Undefined)
+    is_binnned = t.Bool(t.Undefined)
```

Comment:
The default comes in the `init` on line 225. Shouldn't that be enough?

Reply:
Yes, this is working too. I suggested this, thinking that it would be more explicit, but actually, it can be misleading too, if in the future, it is not consistent with the default value defined in in `__init__`...